draft-ietf-quic-transport.md

```diff
@@ -1351,6 +1355,14 @@ connection when they move networks.  This includes state that can be hard to
 recover such as outstanding requests, which might otherwise be lost with no easy
 way to retry them.
 
+An endpoint that receives packets that contain a source IP address and port that
```

Comment:
I didn't want to accidentally add multipath here.  The preemptive thing is a fine idea, which I encourage you to put in an issue, though I suspect that we'll want to park it.

BTW, I do want to add multipath.  I'm itching to do multipath, but I hear that there are dragons and we're not done here yet.

The point about reordering is certainly valid.  The new path could be shorter, for instance.  Changes inbound.